src/tests/functional/plugin/cpu/single_layer_tests/shape_ops.cpp

```diff
@@ -1,256 +1,262 @@
-//// Copyright (C) 2018-2022 Intel Corporation
```

Comment:
minor, can be done later
I guess we also need to add test with sequence of dynamic `Reshape` to have test which check that reallocation in inPlace case work correct

Reply:
New reshape chain test has been introduced.